# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/xcdroast/install-path.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.org	Tue Jul 17 23:46:26 2001
+++ ./Makefile	Wed Jan  9 23:42:03 2002
@@ -5,8 +5,8 @@
 
 # Installation prefix directory:
 
-#PREFIX=/usr
-PREFIX=/usr/local
+PREFIX=/usr
+#PREFIX=/usr/local
 PRE_LIBDIR=$(PREFIX)/lib/xcdroast-0.98
 
 # put here the prefix where you want xcdroast to look for 
